api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -233,6 +241,76 @@ public User apply(User user) {
     });
   }
 
+  public User setDataUseAgreementBypassTime(Timestamp bypassTime) {
```

Comment:
I think, instead of adding a whole bunch of new methods here, we could make relatively concise calls to updateUseWithRetries directly from ProfileController using lambdas, e.g. something like:

case "dataUseAgreement":
        user = userService.updateWithRetries(user -> {
          user.setDataUseAgreementBypassTime(valueToSet);
          return user;
        });
        break;

Reply:
@blrubenstein what do you think of this?  i know pulling out those db call wrappers into the UserService was a convention thing. not sure how i feel about having it done differently in just one place